tutorial-template.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Markdown template for quantum tutorials
+description: Template and guidance for writing tutorials for the Microsoft Quantum Development Kit documentation. 
+author: bradben
+ms.author: bradben
+ms.date: 04/01/2020
+ms.topic: article
+uid: microsoft.quantum.tutorial-template
+
+# title: <title should match the H1 title - Google displays this on the search results page. (example) "Tutorial: Write and Simulate Qubit-level programs in Q#">
```

Comment:
Title need only match H1 in intent. Most important: Intent displays in search result without truncating key terms. Also, make sure titles and headings are sentence case.

title: <title states article intent succinctly so it fits on the Google search results page. Follow guidance in https://aka.ms/seo-for-writers-cheat-sheet. Ex.: "Tutorial: Write & simulate qubit-level programs in Q#